src/Installer/Form/SrijanDemoInstallForm.php

```diff
@@ -0,0 +1,100 @@
+<?php
```

Comment:
I believe, it makes more sense to name it that way, like `SrijanDevportalInstallForm`